labgraph/examples/simple_viz_fixed_rate.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+# Built-in imports
+import asyncio
+import time
+from dataclasses import field
+from typing import Any, List, Optional, Tuple, Dict
+
+# Import labgraph
+import labgraph as lg
+
+# Imports required for this example
+import matplotlib.animation as animation
+import matplotlib.axes
+import matplotlib.pyplot as plt
+import numpy as np
+
+from .random_message import RandomMessage
+from .rate import Rate
+
+# Constants used by nodes
+SAMPLE_RATE = 10.0
+NUM_FEATURES = 100
+WINDOW = 2.0
+REFRESH_RATE = 2.0
+
+
+# ================================= NOISE GENERATOR ====================================
+
+
+# Configuration for NoiseGenerator, see docs: Lifecycles and Configuration
+class NoiseGeneratorConfig(lg.Config):
+    sample_rate: float  # Rate at which to generate noise
+    num_features: int  # Number of features to generate
+
+
+# A data source node that generates random noise to a single output topic
+class NoiseGenerator(lg.Node):
+    OUTPUT = lg.Topic(RandomMessage)
+    config: NoiseGeneratorConfig
+
+    # A publisher method that produces data on a single topic
+    @lg.publisher(OUTPUT)
+    async def generate_noise(self) -> lg.AsyncPublisher:
+
+        rate = Rate(self.config.sample_rate) 
+        
+        while True:
+            
```

Comment:
Redundant space. Consider using Flake8 for styling. https://flake8.pycqa.org/en/latest/
TODO for Jimmy: Flake8 Style Enforcement.